api/docs/installation/localvir.md

```diff
@@ -52,6 +52,24 @@ $ source badgeyay/bin/activate
 ```sh
```

Comment:
Please reformat the commands as steps (Like Step 1, 2 and so on ..) and remove the unnecessary text as it should only the Installation steps.